nixos/modules/image/repart-image.nix

```diff
@@ -0,0 +1,69 @@
+{ name, fileSystems, partitions, split, seed, definitionsDirectory }:
+
+{ lib
+, runCommand
+, python3
+, black
+, ruff
+, mypy
+, systemd
+, fakeroot
+, util-linux
+, dosfstools
+, mtools
+, e2fsprogs
+, squashfsTools
+, erofs-utils
+, btrfs-progs
+, xfsprogs
+}:
+
+let
+  amendRepartDefinitions = runCommand "amend-repart-definitions.py"
+    {
+      nativeBuildInputs = [ python3 black ruff mypy ];
```

Comment:
```suggestion
      nativeBuildInputs = [ python3 black ruff ];
```

Reply:
I'll defer that to a separate PR, just trying to fix the cross-compilation use-case for now